client-ts/signalr-protocol-msgpack/package.json

```diff
@@ -43,6 +43,7 @@
     "@aspnet/signalr": "^1.0.0-preview1"
   },
   "dependencies": {
+    "@aspnet/signalr": "file:../signalr",
```

Comment:
I would say add a comment about why this is ugly but JSON + comments = 🤷‍♂️

Reply:
ack, that's not even supposed to be there :). It's because I had the script written wrong originally (so it installed into the parent package.json instead of initing a new one and installing into that).